frontend/src/pages/distributedWorkloads/global/projectMetrics/sections/DWSectionCard.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Card, CardTitle, CardHeader, Divider } from '@patternfly/react-core';
+
+export const DWSectionCard: React.FC<{ title: string; content: React.ReactNode }> = ({
+  title,
+  content,
+}) => (
+  <Card isFullHeight className="dw-section-card">
```

Comment:
Don't use classNames this way -- you told me it was used in tests. The goal is to use data-testid instead.

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/2710